ocp_resources/pipeline.py

```diff
@@ -35,6 +35,7 @@ def __init__(
             name=name,
             namespace=namespace,
             client=client,
+            yaml_file=yaml_file,
```

Comment:
`yaml_file` must be passed to `super`, also `name`, `namespace` and `client`needs to be removed and be part of `kwargs`